.eslintrc.js

```diff
@@ -1,10 +1,24 @@
 module.exports = {
   extends: ['vtex'],
   overrides: [
-    // Use vtex-react preset for .tsx files
     {
-      files: '*.tsx',
-      extends: ['vtex-react'],
+      files: [
+        // all tsx files
+        '*.tsx',
+        // gatsby config files (gatsby preset takes care of each variant)
+        'gatsby-*.*',
+        // generic components/hooks path
+        'components/**/*',
+        'hooks/**/*',
```

Comment:
why hooks ?

Reply:
hooks should be linted with the `vtex-react` preset, not the base `vtex` one. Or I understood you wrong?

Reply:
yes, but question is more like: "Should we support hooks in custom files or all hooks should be on its own file called `use*` ?"

Reply:
Oh, that's an interesting decision to be made. I'd say to have only `use*` files, but we can leave these globs for the time being and create an issue to refactor each hook to its own file. What do you think?

Reply:
#413